### PR TITLE
docs: added migration from oss

### PR DESCRIPTION
**Pull request by exu**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: docs/docs/testkube-cloud/installing-agent.md [hunk 1]
```diff
@@ -32,8 +32,16 @@ You will need *Helm* installed and `kubectl` configured with access to your Kube
 
 ## Validating the Installation 
 
-Testkube Cloud will notify if the installation is successful. A green indicator means that your cluster was able to connect to Testkube Cloud.
+Testkube Cloud will notify if the installation is successful. 
+
+* A green indicator means that your cluster was able to connect to Testkube Cloud.
+* A red indicator indicates that the Testkube Agent can't connect to the Testkube Cloud API (Testkube need some time to establish connection max 2-3 minutes).
```
Comment (jfermi) [inline docs/docs/testkube-cloud/installing-agent.md:38]:
```suggestion
* A red indicator indicates that the Testkube Agent can't connect to the Testkube Cloud API (Testkube needs some time to establish a connection, max time is 2-3 minutes).
```

File: docs/docs/testkube-cloud/managing-cli-context.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+## Changing the Testkube context to cloud one
```
Comment (jfermi) [inline docs/docs/testkube-cloud/managing-cli-context.md:1]:
```suggestion
## Changing the Testkube Context to Cloud
```

File: docs/docs/testkube-cloud/managing-cli-context.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+To set context to could one, testkube CLI tool need to have access, so first you'll need to create API token with
```
Comment (jfermi) [inline docs/docs/testkube-cloud/managing-cli-context.md:3]:
```suggestion
To set the Testkube context to Cloud, the Testkube CLI tool needs to have access, so first you'll need to create an API token with 
```

File: docs/docs/testkube-cloud/managing-cli-context.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+## Changing the Testkube context to cloud one
+
+To set context to could one, testkube CLI tool need to have access, so first you'll need to create API token with 
+valid access rights.
+
+You can create token:
```
Comment (jfermi) [inline docs/docs/testkube-cloud/managing-cli-context.md:6]:
```suggestion
You can create a token:
```

File: docs/docs/testkube-cloud/managing-cli-context.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+* with "admin" access rights (access to all environments)
```
Comment (jfermi) [inline docs/docs/testkube-cloud/managing-cli-context.md:8]:
```suggestion
* with "admin" access rights (access to all environments). 
```

File: docs/docs/testkube-cloud/managing-cli-context.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+![admin-token](https://user-images.githubusercontent.com/30776/229772185-01f1e466-b04d-4c6d-9d5c-e4464d651177.png)
+
+* with particular role for given environments
+
+![roles-for-token](https://user-images.githubusercontent.com/30776/229772310-64bda85d-57a8-47b7-a68b-2625089724f8.png)
+
+
+
+Now when your token is there you're ready to change Testkube CLI context:
```
Comment (jfermi) [inline docs/docs/testkube-cloud/managing-cli-context.md:18]:
```suggestion
When your token is there, you're ready to change the Testkube CLI context: 
```

File: docs/docs/testkube-cloud/managing-cli-context.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+
+Now when your token is there you're ready to change Testkube CLI context: 
+
+![setting-context](https://user-images.githubusercontent.com/30776/229771159-4415aa74-70bb-4684-9511-449d0779b483.png)
+
+
+Changing Testkube context to kubeconfig based
+
+When you want to get back to using Testkube CLI with your local OSS Testkube cluster just set context to kubeconfig based:
```
Comment (jfermi) [inline docs/docs/testkube-cloud/managing-cli-context.md:25]:
```suggestion
When you want to return to using Testkube CLI with your local OSS Testkube cluster,  set the context to kubeconfig-based: 
```

File: docs/docs/testkube-cloud/transition-from-oss.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+# Transition from Testkube OSS
+
+To migrate Testkube OSS to Cloud you need to install Testkube in Cloud Agent mode. Testkube Cloud Agent is the Testkube engine for managing test runs into your cluster. It sends data to Testkubes Cloud Servers. It's main responsibility is to manage test workloads and to get insight into Testkube resources stored in the cluster.
```
Comment (jfermi) [inline docs/docs/testkube-cloud/transition-from-oss.md:3]:
```suggestion
To migrate Testkube OSS to Cloud you need to install Testkube in Cloud Agent mode. Testkube Cloud Agent is the Testkube engine for managing test runs into your cluster. It sends data to Testkube's Cloud Servers. Its main responsibility is to manage test workloads and to get insight into Testkube resources stored in the cluster.
```

File: docs/docs/testkube-cloud/transition-from-oss.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+## Installing the Agent
+
+Please follow the [install steps](installing-agent.md) to get started using the Testkube Agent.
+
+```
+helm repo add kubeshop https://kubeshop.github.io/helm-charts ; helm repo update && helm upgrade --install --create-namespace testkube kubeshop/testkube --set testkube-api.cloud.key=tkcagnt_aaaaaaaaaaaaaaaaaaaaakey --set testkube-api.minio.enabled=false --set mongodb.enabled=false --namespace testkube
+```
+
+WARNING! Please keep in mind that default install will REMOVE existing MongoDB, Minio and Dashboard pods!
```
Comment (jfermi) [inline docs/docs/testkube-cloud/transition-from-oss.md:14]:
```suggestion
WARNING! Please keep in mind that the default install will REMOVE existing MongoDB, MinIO and Dashboard pods!
```

File: docs/docs/testkube-cloud/transition-from-oss.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+To keep them set below options to true (3 values for MongoDB, MinIO, Dashboard):
```
Comment (jfermi) [inline docs/docs/testkube-cloud/transition-from-oss.md:16]:
```suggestion
To keep the pods, set the below options to true (3 values for MongoDB, MinIO, Dashboard):
```

File: docs/docs/testkube-cloud/transition-from-oss.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+```sh
+ --set testkube-api.minio.enabled=true --set mongodb.enabled=true --set testkube-dashboard.enabled=true
+```
+
+## Setting the Testkube CLI context to the agent mode
```
Comment (jfermi) [inline docs/docs/testkube-cloud/transition-from-oss.md:21]:
```suggestion
## Setting the Testkube CLI Context to Agent Mode
```

File: docs/docs/testkube-cloud/transition-from-oss.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+Please follow the [context management](managing-cli-context.md) to configure your Testkube CLI in Cloud mode.
+
+
+## Migrating the Testkube Resources
+
+Currently there is no automatic migration tool for existing Testkube OSS resources. But we have plan for it in incoming releases.
```
Comment (jfermi) [inline docs/docs/testkube-cloud/transition-from-oss.md:28]:
```suggestion
Currently there is no automatic migration tool for existing Testkube OSS resources. This is planned for coming releases.
```

**Comment by exu** [comment]
@jfermi commited all of them :) thanks!

**Comment by exu** [comment]
@jfermi added some new pages, could you revisit them please?

**Review by aabedraba — APPROVED** [review]

**Pull request merged**